modules/ting_field_search/ting_field_search.facets.inc

```diff
@@ -23,21 +23,22 @@ function _ting_field_search_alter_facets() {
  */
 function ting_field_search_form_ding_facetbrowser_form_alter(&$form, &$form_state) {
   if ($profile = _ting_field_search_alter_facets()) {
-    // Get the facets returned from the datawell.
-    // TODO BBS-SAL: Implement proper facet support.
+    // Get the facets returned from the search provider.
     $search_result = ting_search_current_results();
-    if (empty($search_result->openSearchResult->facets)) {
+    if (empty($search_result) || empty($search_result->getFacets())) {
       return;
     }
-    $facets = $search_result->openSearchResult->facets;
+
+    // Until the facet-browser can be switched to specifying facets by their
+    // id, we need to be able to look them up by name.
+    $facets = $search_result->getFacets();
```

Comment:
How does the comment relate to the code it is preceeding?

Reply:
Yet another leftover from a refactor - good catch.